data/views/node.2.0.json

```diff
@@ -1,5 +1,6 @@
 {
     "autoDiscover": "<%=autoDiscover%>",
+    "associatedServer":"<%=associatedServer%>",
```

Comment:
From the property name, I think it can be moved into a new kind of relation, rather than create a new property in node document.